src/backend/distributed/planner/insert_select_planner.c

```diff
@@ -1355,3 +1449,224 @@ InsertSelectResultIdPrefix(uint64 planId)
 
 	return resultIdPrefix->data;
 }
+
+
+/*
+ * RelableTargetEntryList relabels select target list to have matching names with
+ * insert target list.
+ */
+static void
+RelableTargetEntryList(List *selectTargetList, List *insertTargetList)
```

Comment:
Moved as is from insert_select_executor.

Reply:
maybe Relable -> Relabel